mava/utils/total_timestep_checker.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import jax
+from colorama import Fore, Style
+from omegaconf import DictConfig
+
+
+def check_total_timesteps(config: DictConfig) -> DictConfig:
+    """Check if total_timesteps is set, if not, set it based on the other parameters"""
+    n_devices = len(jax.devices())
+    if config.system.total_timesteps is None:
+        config.system.total_timesteps = (
+            n_devices
+            * config.system.num_updates
+            * config.system.rollout_length
+            * config.system.update_batch_size
+            * config.arch.num_envs
+        )
+    else:
+        config.system.num_updates = (
```

Comment:
I suggest updating the number of evaluations as well since this will raise an error if we insert a `total_timesteps` that gives `num_updates `lesser than the `num_evaluation`.

Reply:
Suggestion: 
```python
if num_updates < num_evaluatio:
 num_evaluation = random.randint(5,num_updates)
```

Reply:
I think this is up to the user, changing one variable is enough else it will raise many issues and less control of the vars to the users